lib/_http_outgoing.js

```diff
@@ -558,9 +551,7 @@ OutgoingMessage.prototype.getHeaders = function getHeaders() {
   const headers = this[kOutHeaders];
   const ret = ObjectCreate(null);
   if (headers) {
-    const keys = ObjectKeys(headers);
-    for (var i = 0; i < keys.length; ++i) {
-      const key = keys[i];
+    for (const key of ObjectKeys(headers)) {
```

Comment:
We should not make this change for performance reasons.

Reply:
resolved in https://github.com/nodejs/node/pull/30958/commits/17049d56621869d8776a922d5e56b6c5ec35be7b